Pod/Classes/WPMediaPickerOptions.h

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+#import "WPMediaCollectionDataSource.h"
+
+@interface WPMediaPickerOptions: NSObject
+
+/**
+ If set the picker will show a cell that allows capture of new media, that can be used immediatelly
+ */
+@property (nonatomic, assign) BOOL allowCaptureOfMedia;
```

Comment:
Regarding the allowCaptureOfMedia, I'm not sure we should change it we are not only showing the CaptureCell or preview we are actually allowing users to capture new assets to the datasource.